#!/usr/bin/env python
# field of view configuration

PACKAGE="cob_3d_mapping_semantics"
#import roslib; roslib.load_manifest(PACKAGE)

from math import pi

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("target_frame_id", str_t, 1, "Target frame for table extraction", "/map")
gen.add("tilt_angle", double_t, 3, "maximum tilt angle in degrees", 5, 0, 45)
gen.add("height_min", double_t, 3, "minimum height", 0.5, 0, 2)
gen.add("height_max", double_t, 3, "maximum height", 1.8, 0, 2)
gen.add("area_min", double_t, 3, "minimum area", 0.3, 0, 5)
gen.add("area_max", double_t, 3, "maximum area", 3, 0, 10)

exit(gen.generate(PACKAGE, "dynamic_reconfigure_node", "table_extraction_node"))
